### PR TITLE
Api 32569 null handling currently va employee

### PR DESCRIPTION
**Pull request by mchristiansonVA**
## Summary

- Updates 526 schema to indicate currentVaEmployee is a required field.
- Adjusts handling for diagnosticCode so that it's removed from the EVSS docker mapping in cases where the value is null.

## Related issue(s)
[API-32569](https://jira.devops.va.gov/browse/API-32569)

## Testing done
- Local testing with Postman

## What areas of the site does it impact?
modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json
modules/claims_api/app/swagger/claims_api/v2/production/swagger.json
modules/claims_api/config/schemas/v2/526.json
modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Comment by stiehlrod** [comment]
@mchristiansonVA I am able to send a request without the `currentVaEmployee` attribute. Shouldn't I get a 422 if it is a required field?

**Comment by mchristiansonVA** [comment]
> @mchristiansonVA I am able to send a request without the `currentVaEmployee` attribute. Shouldn't I get a 422 if it is a required field?

Good catch, thanks! Updated schema and re-generated swagger; endpoint correctly responds with a 422 and notes missing required key `currentVaEmployee` in error message.

File: modules/claims_api/config/schemas/v2/526.json [hunk 1]
```diff
@@ -136,10 +139,9 @@
           }
         },
         "currentVaEmployee": {
-          "type": ["boolean", "null"],
+          "type": ["boolean"],
```
Comment (rockwellwindsor-va) [inline modules/claims_api/config/schemas/v2/526.json:142]: I think this should change to just `"type": "boolean"` , right? No array there anymore.

**Review by rockwellwindsor-va — CHANGES_REQUESTED** [review]
I think it would be worth it to add a test for this.  There is one testing for non boolean values on the `currentVaEmployee` on line 499 in the request spec, something under that just locking in this updated requirement would help I think.

File: modules/claims_api/config/schemas/v2/526.json (hunk 1 above)
Comment (mchristiansonVA) [reply modules/claims_api/config/schemas/v2/526.json]: Good catch, thanks!

**Review by rockwellwindsor-va — APPROVED** [review]

**Pull request merged**